Maps section completed. (#996)
Esri/arcgis-runtime-samples-qt: ArcGISRuntimeSDKQt_CppSamples/Maps/OpenMapUrl/README.md

# Open Map (URL)

Display a web map.

![](screenshot.png)

## Use case

> For displaying web maps stored on ArcGISOnline (e.g. soil survey results, as demonstrated in this sample).

Reviewer: ```suggestion
For displaying web maps stored on ArcGIS Online (e.g. soil survey results, as demonstrated in this sample).
```

## How to use the sample

A web map can be selected from the drop-down list. On selection the web map displays in the map view.

## How it works

1. Create a `Portal`.
2. Create a `PortalItem` using the Portal and the web map ID: `new PortalItem(portal, ID)`.
3. Create a `Map` using the portal item.

> 4. Set the map to the `MapView`.

Author: ```suggestion
1. Create a URL comprised of the Organization's Portal URL and the webmap of interest.
2. Create a `Map` using the URL.
3. Set the map to the `MapView`.
```

Author: I wonder why the Qt implementation of this was different from the design.

## Relevant API

* Map
* MapView
* Portal

> * PortalItem

Author: ```suggestion
```
